tests/mapper/test_node_mappers.py

```diff
@@ -926,3 +926,34 @@ def test_generator_methods(self):
             ppnp_sparse_failed = True
 
         assert ppnp_sparse_failed
+
+
+@pytest.mark.parametrize("sparse", [True, False])
+def test_corrupted_full_batch_generator(sparse):
+
+    G, feats = create_graph_features()
+    nodes = G.nodes()
+    node_features = pd.DataFrame.from_dict(
+        {n: f for n, f in zip(nodes, feats)}, orient="index"
+    )
+    G = StellarGraph(G, node_type_name="node", node_features=node_features)
+
+    generator = CorruptedFullBatchNodeGenerator(G, sparse=sparse)
+
+    gen = generator.flow(G.nodes())
+
+    [features, shuffled_feats, *_], targets = gen.__getitem__(0)
+
+    assert features.shape == shuffled_feats.shape
+
+    # check shuffled_feats are feats
+    assert not np.array_equal(features, shuffled_feats,)
```

Comment:
There's quite a few instances of trailing commas on a single line in this PR (search for `,)` and `,]` to find others); is this on purpose?

```suggestion
    assert not np.array_equal(features, shuffled_feats)
```

Reply:
I had a fair bit of multiline stuff that I refactored (or maybe black did) but forgot to cut out the trailing commas, will delete